### ISSUE TITLE
Reframing & re-defining our stakeholders

### ISSUE THREAD

**Issue by lscheske** [publiccodenet/about#660]
We are currently working and collaborating with various "stakeholders" (organizations, vendors, civil society) to advance our mission. The majority of these are public organizations that are in the ‘membership pipeline’. The past year has shown that getting public organizations to sign a membership contract to join our association is a long and bureaucratic process. We are investing valuable resources into this ‘pre-membership’ phase through codebase stewardship and advisory services. In order to better reflect our current process and make the most of these relationships, we would like to propose new definitions for the various stakeholders we engage with. This will allow for clearer communication in our marketing assets (e.g. website), and make it easier for us to talk about how we collaborate with different stakeholders. 

_By agreeing on this issue, we can unblock #619 and proceed with developing more comms material for publiccode.net_

We would like to (losely) define the following stakeholders: 

**Members**:
Members are wholly publicly owned organizations that are officially part of, and able to govern, the Foundation for Public Code vereniging. They pay membership dues, vote in the general assembly, and help shape our processes by partaking in the governance and finance of our organization.
Formal criteria: signed a membership contract, and abide by our articles of association, which includes paying yearly membership dues and voting in the general assembly.

**Partners**:
Partners are publicly owned organizations with whom we work closely together. They may include organizations that want to become members in the future; or organizations that do not want to become members, but are interested in supporting and participating in our community.  In the previous model, these organizations would have been those who had signed the Expression of Interest. Going forward, we will simply describe those who we are working with and are not yet Members as Partners [after they give their permission via email]. We will retire the Expression of Interest document altogether.

**Affiliates**:
Affiliates are non-profit organizations whose missions align with ours, and whom we collaborate with.

**Vendors**:
Vendors are private companies that are part of the public code ecosystem. They may be active participants in our community, or directly contribute to public code projects by supplying services.

**Sponsors**:
Sponsors are any organization that provides financial support to the Foundation for Public Code. e.g. by sponsoring events, or structural funding.

**Comment by Ainali** [comment]
Should affiliates also agree like partners, or can we choose to call organizations we think are aligned as affiliates without their consent?

**Comment by ElenaFdR** [comment]
Are these all mutually exclusive categories? Would an organization ever be both a partner and a sponsor? Only some sponsors are also affiliates?

**Comment by ElenaFdR** [comment]
To be clear: I suggest mutually exclusive categories, as much as possible, in order to make it conceptually simple for someone external to understand what an org's relationship to us is.

**Comment by lscheske** [comment]
@Ainali, I think we can have a similar less formal e-mail agreement as what we propose with Partners - any other ideas are welcome too! A kind of Memorandum of Understanding would also be an option, but may be perceived as too formal. 

@ElenaFdR, good point - I see all of these as mutually exclusive except for the Sponsor category. We thought that Partners/Affiliates that are able and willing to give us money may do so without becoming members. So that would make this the only slightly different category. Partners and Members are linked because some Partners could be on a trajectory to become Members, whereas others may not ever become members.

**Comment by bvhme** [comment]
Thanks for this elaborate proposal :)

This issue proposes some large and fundamental changes to the way we operate. They also seem not all dependent on each other. I would like to see some of them discussed on their own merit and not all as one. So, perhaps we can spend some time peeling these apart to look at them.

I hope that our effort to get more testimonials, case studies, and storytelling about our work on our website can at least get us started on those underlying issues and hopefully move that forward whilst we consider this.

Apart from that, I'm interested in the issues behind this proposal.

Not being able to follow along makes me quite critical, for instance:

* I recognize it is important to be able to clearly communicate what's going on and show things of, but am skeptical that the solution is in agreements, systems and process that we dedicate ourselves and others to.
* I understand there is a need to communicate more with the outside world about our relations, however, that is its own issue that can be addressed without introducing more process.
* If it more about is clarity in communications, I'm also wondering if there are things we can achieve outside of creating another formal class of relationships.
* If it is about dealing with the long process of building membership, I don't see how constricting our space for interaction in the way that they/we need will help.

To respond to @ElenaFdR, we've decided to keep these as completely separate things so that can be what we need them to be in order to succeed. So as labels and not categories/instances. This because these relationships have different missions and goals and are not easily compared. To me, it makes little sense to make these uniform and tied together if that means we can't shape and form them as we need.

And, @Ainali since Affiliations don't necessarily have a lot of things we can do in them to talk about they need some kind of mutual acknowledgment to be useful. We have an MOU with [Agile and Open Smart Cities](https://oascities.org/) and [OW2](https://www.ow2.org/) for instance.

**Comment by nebster** [comment]
This is an issue that gets me back into github!  I am still quite keen on moving to the new terminology @bvhme @lscheske and I had previously agreed on.  I would especially respond that this change, namely the move to call potential members we are working with "partners" in our communications materials and online presence, does not in fact involve any more process than asking them for this permission in an email.  I am in agreement that we can achieve this incrementally by first asking for testimonials and then moving to ask to include those from whom we receive them in a list of partners with logos.

**Comment by nebster** [comment]
@ElenaFdR @bvhme I would also suggest that these labels are not in face mutually exclusive because organizations can in fact fall into more than one category, so ease of description is less relevant if it is indeed incorrect.

**Comment by bvhme** [comment]
When splitting this issue, these already exist:

Existing issue for defining Affiliates: https://github.com/publiccodenet/about/issues/496
Existing issue for defining Partner: https://github.com/publiccodenet/about/issues/499

**Comment by lscheske** [comment]
Note: we will define the stakeholders we work with for our internal, working purposes. There will be no formal agreements for any stakeholders other than members at this point. 

I will make separate issues for these definitions so that they can be taken up in our glossary.

**Comment by lscheske** [comment]
Largely closed by #687 #688 #689

Further terms (vendors, sponsors) not yet defined because the use cases don't yet exist.
